service-discovery/kubernetes/src/test/java/io/smallrye/stork/servicediscovery/kubernetes/KubernetesServiceDiscoveryTest.java

```diff
@@ -67,6 +67,33 @@ void shouldGetServiceFromK8sDefaultNamespace() {
                 "10.96.96.232", "10.96.96.233");
     }
 
+    @Test
+    void shouldDiscoverServiceWithSpecificName() {
+
+        TestConfigProvider.addServiceConfig("svc", null, "kubernetes",
+                null, Map.of("k8s-host", k8sMasterUrl, "application", "rest-service"));
+        Stork stork = StorkTestUtils.getNewStorkInstance();
+
+        String serviceName = "svc";
+
+        setUpKubernetesService("rest-service", null, "10.96.96.231", "10.96.96.232", "10.96.96.233");
```

Comment:
Maybe also add a service with a different name, to test that only the ones with the specified name are returned

Reply:
ah, yes, good idea. I will do it